app/controllers/concerns/saml_idp_logout_concern.rb

```diff
@@ -25,7 +25,9 @@ def handle_valid_sp_remote_logout_request(user_id)
     # Remotely invalidate the user's current session, see config/initializers/session_limitable.rb
     User.find(user_id).update!(unique_session_id: nil)
 
+    # rubocop:disable Rails/RenderInline
     render inline: logout_response, content_type: 'text/xml'
+    # rubocop:enable Rails/RenderInline
```

Comment:
the new cop prefers using a template instead of `render inline:` however in this case for SAML, we really don't have a good template, but I figured the cop was a good one to keep